packages/edit-site/src/components/global-styles/gradients-palette-panel.js

```diff
@@ -45,7 +45,16 @@ export default function GradientPalettePanel( { name } ) {
 		'color.defaultGradients',
 		name
 	);
-	const [ duotonePalette ] = useSetting( 'color.duotone' ) || [];
+
+	const [ defaultDuotone ] = useSetting( 'color.duotone.default' ) || [];
+	const [ themeDuotone ] = useSetting( 'color.duotone.theme' ) || [];
+	const [ defaultDuotoneEnabled ] = useSetting( 'color.defaultDuotone' );
+
+	const duotonePalette = [
+		...( themeDuotone || [] ),
+		...( defaultDuotone && defaultDuotoneEnabled ? defaultDuotone : [] ),
+	];
```

Comment:
In the duotone.js hook, the palette also adds `'color.duotone.custom'` first in the list. That list can't be changed in the editor right now, but it would be nice to have them included here for consistency in case they've been updated manually.

https://github.com/WordPress/gutenberg/blob/4f7c19e37d55bb6a419cc8eca94c634f1e71852e/packages/block-editor/src/hooks/duotone.js#L58-L74

Reply:
Thanks for the review! I think that makes sense.

✅ I have added a custom duotone.
✅ I confirmed that the testing instructions I described is expected results as well.